enos/enos-scenario-smoke.hcl

```diff
@@ -167,35 +183,80 @@ scenario "smoke" {
   }
 
   step "verify_vault_unsealed" {
-    module = module.vault_verify_unsealed
-    depends_on = [
-      step.create_vault_cluster,
-    ]
+    module     = module.vault_verify_unsealed
+    depends_on = [step.create_vault_cluster]
 
     providers = {
       enos = local.enos_provider[matrix.distro]
     }
 
     variables {
-      vault_instances  = step.create_vault_cluster.vault_instances
-      vault_root_token = step.create_vault_cluster.vault_root_token
+      vault_install_dir = local.vault_install_dir
+      vault_instances   = step.create_vault_cluster.vault_instances
+      vault_root_token  = step.create_vault_cluster.vault_root_token
     }
   }
 
   step "verify_raft_auto_join_voter" {
-    skip_step = matrix.backend != "raft"
-    module    = module.vault_verify_raft_auto_join_voter
-    depends_on = [
-      step.create_vault_cluster,
-    ]
+    skip_step  = matrix.backend != "raft"
+    module     = module.vault_verify_raft_auto_join_voter
+    depends_on = [step.create_vault_cluster]
+
+    providers = {
+      enos = local.enos_provider[matrix.distro]
+    }
+
+    variables {
+      vault_install_dir = local.vault_install_dir
+      vault_instances   = step.create_vault_cluster.vault_instances
+      vault_root_token  = step.create_vault_cluster.vault_root_token
+    }
+  }
+
+  step "verify_replication" {
+    skip_step  = matrix.artifact_type != "package"
```

Comment:
Why don't we validate replication when testing with other artifact types?

Reply:
RE: this and your other comment about skipping modules -- I kept the same structure that was in the original smoke tests. There were modules that were only being used for the `package_manager` scenario so I left it that way here too. But if we think it makes sense to run all of these modules for all artifact types, we can certainly do that.

Reply:
As a conclusion to the question above, we agreed to include all of the same modules for both artifact types, rather than maintain the delineation that was in the old smoke tests.